clouddriver-artifacts/src/main/java/com/netflix/spinnaker/clouddriver/artifacts/bitbucket/BitbucketArtifactCredentials.java

```diff
@@ -38,6 +42,18 @@
     this.name = account.getName();
   }
 
+  @Override
+  protected Headers getHeaders(BitbucketArtifactAccount account) {
+    Headers.Builder headers = new Headers.Builder();
+    Optional<String> token = account.getTokenAsString();
+    if (token.isPresent()) {
+      headers.set(AUTHORIZATION, "Bearer " + token.get());
```

Comment:
Does Authorization need to be capitalized?

Reply:
I used a constant i.e.
`public static final String AUTHORIZATION = "Authorization";`
by importing
`import static org.springframework.http.HttpHeaders.AUTHORIZATION;`